src/tests/ftest/recovery/pass_1.py

```diff
@@ -17,18 +17,104 @@ class Pass1Test(TestWithServers):
     :avocado: recursive
     """
 
+    def test_run_checker(self, inconsistency=None, policies=None):
```

Comment:
Helper methods should not start with `test_` because that's the naming convention for actual tests.

Also, this function should document the args:
```
Args:
  inconsistency (<type>, optional): what this is. Defaults to None.
  policies(<type>, optional): what this is. Defaults to None.
```

Reply:
Also we should add Returns:
```
Returns:
    list: Errors.
```